cmd/operator/deploy/crds/monitoring.googleapis.com_operatorconfigs.yaml

```diff
@@ -105,31 +105,15 @@ spec:
             type: string
           managedAlertmanager:
             default:
-              config:
-                key: config.yaml
-                name: managed-alertmanager-config
+              configSecret: managed-alertmanager-config
             description: ManagedAlertmanager holds information for configuring the
               managed instance of Alertmanager.
             properties:
-              config:
-                description: Config refers to the Secret in the gmp-public namespace
-                  that holds the managed Alertmanagers config file. This secret is
-                  copied to gmp-system by the operator.
-                properties:
-                  key:
-                    description: The key of the secret to select from.  Must be a
-                      valid secret key.
-                    type: string
-                  name:
-                    description: 'Name of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names
-                      TODO: Add other useful fields. apiVersion, kind, uid?'
-                    type: string
-                  optional:
-                    description: Specify whether the Secret or its key must be defined
-                    type: boolean
-                required:
-                - key
-                type: object
+              configSecret:
```

Comment:
nit: `singleKeyConfigSecret`.

Reply:
do you think it's okay to leave `singleKey` out of the name since that's the only option right now? If we do add a multiKey option later, it would make sense to differentiate in the name then. but for simplicity I think the name is good as-is, and just document that it needs to have only 1 key. wdyt?

Reply:
So in the future case we'll provide `configSecret` and `multiKeyConfigSecret`? Sounds fine to me.

Reply:
yup that's what I was thinking